src/main/java/jenkins/plugins/clangscanbuild/publisher/ClangScanBuildPublisherDescriptor.java

```diff
@@ -41,10 +40,7 @@ public String getDisplayName() {
 
 	@Override
 	public boolean isApplicable( @SuppressWarnings("rawtypes") Class<? extends AbstractProject> jobType ){
-		if( !FreeStyleProject.class.isAssignableFrom( jobType ) ){
-			System.err.println( "Clang scan-build ERROR: Expected FreeStyleProject but was: " + jobType + " at Publisher Descriptor" );
-		}
-		return AbstractProject.class.isAssignableFrom( jobType );
+		return true;
```

Comment:
A large part of plugins are returning yes. Only for few cases it is important to not allow the usage in some specific projects types. The specificity of this plugin is to be useful for iOS/XCode builds (thus on MacOs slaves only) but like the Xcode plugin you should be able to use it in FreeStyle, Matrix and others plugins. I agree that some integration tests to validate that it won't break various jobs types may be useful but I don't know how we could write them